gnovm/stdlibs/crypto/ed25519/ed25519.gno

```diff
@@ -0,0 +1,8 @@
+package ed25519
+
+// Verify returns true if the signature is valid for the message and public key.
+func VerifyED25519(publicKey []byte, message []byte, signature []byte) bool {
+	return verifyED25519(publicKey, message, signature)
+}
+
+func verifyED25519(publicKey []byte, message []byte, signature []byte) bool // injected
```

Comment:
Sorry for the back and forth on this issue, there was a bit of confusion :dizzy_face: 

Since the Go function is called [Verify](https://pkg.go.dev/crypto/ed25519#Verify), this one should also be Verify (ie, it was correct before!). Apologies again :pray: